api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -358,17 +358,37 @@ message SendLogsRequest {
   }
 }
 
-// Message to encapsulate setting the log level.
-message SetLogLevel {
-  // The log level to be set in the operator.
-  LogLevel level = 1;
-}
+//// Message to encapsulate setting the log level.
+//message SetLogLevel {
+//  // The log level to be set in the operator.
+//  LogLevel level = 1;
+//}
 
 // Message to encapsulate the response for sending logs.
 message SendLogsResponse {
+  //  oneof response {
+  //    // set_log_level sets the operator's log level to the given level.
+  //    SetLogLevel set_log_level = 1;
+  //  }
+}
```

Comment:
Delete code instead of commenting it.
Trust GitHub to not lose the history of changes.
```suggestion
message SendLogsResponse {}
```